onnxmltools/convert/common/_apply_operation.py

```diff
@@ -438,3 +439,46 @@ def apply_softmax(scope, input_name, output_name, container, operator_name=None,
 def apply_normalization(scope, input_name, output_name, container, operator_name=None, axis=1, p=2):
     name = _create_name_or_use_existing_one(scope, 'LpNormalization', operator_name)
     container.add_node('LpNormalization', input_name, output_name, name=name, p=p, axis=axis)
+
+def apply_crop_height_width(scope, input_name, output_name, container, operator_name=None,
+        top_border=0, bottom_border=0, left_border=0, right_border=0):
+    name = scope.get_unique_operator_name('CropHeightWidth')
+    if container.target_opset < 9:
```

Comment:
double confirm: <9 or <=9?

Reply:
My understanding is that Opset 9 still has experimental ops, and that they will be deprecated in Opset 10 (ONNX v1.5) -- https://github.com/onnx/onnx/issues/1752

Reply:
DynamicSlice exists since onnx-1.3, so we can use it for opset = 9, right?